doc/user_guide/_docs/A03-constant.md

```diff
@@ -1,14 +1,12 @@
 ---
-title: "Constant"
+title: "Constants"
```

Comment:
I am not sure if changing the word here will break the link in https://github.com/intel/rohd-website/blob/main/docs/_data/navigation.yml.

Reply:
I was under the impression that as long as the permalink didn't change it would be safe?

Reply:
Ya, I think this should be okay.